nav2_controller/plugins/simple_goal_checker.cpp

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
Can I assume these `cpp` files are for-batum from the dwb versions?

Reply:
Yes